Resources/translations/messages.de.xliff

```diff
@@ -90,6 +90,49 @@
         <source>web.view.mail.not.available.title</source>
         <target>Email nicht gefunden!</target>
       </trans-unit>
+
+      <trans-unit id="3dtcx8jdffgdsfd2sthvj119c9pmhttuvzdy57j9" resname="email.dashboard.label">
+        <source state="new">email.dashboard.label</source>
```

Comment:
The `state="new"` is supposed to be on the `target` and not on the `source`. See https://github.com/azine/AzineMailgunWebhooksBundle/pull/18#discussion_r149360296
Can you please move it everywhere on this PR. Thanks.